app/views/catalog/_show_documents.html.erb

```diff
@@ -44,10 +44,16 @@
 
 <script type="text/javascript">
 $(function() {
+
+  // Wire DataTable for the file list.
+  $('#files-table').DataTable();
```

Comment:
This is it. Once the DataTables plug-in has been added to the project this is all is needed to get pagination on an HTML table (in this case the one with `id="files-table"`)